metagraph/src/annotation/representation/column_compressed/annotate_column_compressed.cpp

```diff
@@ -107,6 +144,44 @@ void ColumnCompressed<Label>::serialize(const std::string &filename) const {
         assert(column.get());
         column->serialize(outstream);
     }
+
+    if (!relation_counts_.size())
+        return;
+
+    outstream.close();
+
+    outstream.open(remove_suffix(filename, kExtension) + kExtension + ".counts",
+                   std::ios::binary);
+    if (!outstream.good())
+        throw std::ofstream::failure("Bad stream");
+
+    uint64_t num_counts = 0;
+    uint64_t sum_counts = 0;
+
+    for (size_t j = 0; j < relation_counts_.size(); ++j) {
+        if (!relation_counts_[j].size()) {
+            sdsl::int_vector<>(bitmatrix_[j]->num_set_bits(), 0, kCountWidth).serialize(outstream);;
+
+        } else if (relation_counts_[j].size() != bitmatrix_[j]->num_set_bits()) {
```

Comment:
This can only happen because of a programming error, right? Wouldn't an `assert` be more appropriate?

Reply:
done